tests/ga/test_extension.py

```diff
@@ -48,7 +48,9 @@
 from tests.protocol.mocks import mock_wire_protocol, HttpRequestPredicates
 from tests.protocol.mockwiredata import DATA_FILE
 from tests.tools import AgentTestCase, data_dir, i_am_root, MagicMock, Mock, patch
-
+from tests.ga.extension_emulator import Actions, ExtensionCommandNames, extension_emulator, \
+    enable_invocations, generate_put_handler 
```

Comment:
I see some unused imports in this file (not related to your changes). Could you please clean those up too since you're modifying this code anyways. Thanks!